hera_mc/tests/test_geo_location.py

```diff
@@ -17,18 +17,66 @@
 
 class TestGeo(TestHERAMC):
 
-    def test_add_part(self):
-        self.test_session.add(geo_location.GeoLocation())
+
+    def test_add_loc(self):
```

Comment:
I'm not sure this test is adding anything, given that your 3rd test does exactly the same thing and then retrieves the value and checks that it is as expected.